src/review-form.ts

```diff
@@ -138,7 +191,7 @@ export class ReviewForm extends LitElement {
         type="submit"
         class="ia-button primary"
         name="submit"
-        value="Submit review"
```

Comment:
QUESTION: Where does the button label get set if the `value` attribute is removed? Or are you just letting it use the default "Submit"?